dso-l2/src/main/java/com/tc/objectserver/entity/ManagedEntityImpl.java

```
@@ -335,6 +336,7 @@ private void invoke(ServerEntityRequest request, EntityMessage message, int conc
       } catch (Exception e) {
         // Wrap this exception.
         EntityUserException wrapper = new EntityUserException(id.getClassName(), id.getEntityName(), e);
+        logger.error("caught exception during invoke ", wrapper);
         request.failure(wrapper);
         throw new RuntimeException(e);
```

should we throw this exception?